scripts/postbuild.js

```diff
@@ -8,7 +9,8 @@ copyFileSync('src/ng-package.schema.json', 'dist/ng-package.schema.json', consta
 mkdirSync('dist/lib/ts/conf', { recursive: true });
 copyFileSync('src/lib/ts/conf/tsconfig.ngc.json', 'dist/lib/ts/conf/tsconfig.ngc.json', constants.COPYFILE_FICLONE);
 
-const packageJson = JSON.parse(readFileSync('package.json'));
+const packageJsonPath = resolve(process.cwd(), 'package.json');
+const packageJson = require(packageJsonPath);
```

Comment:
In the case since we are mutating the manifest, it's best to use `JSON.parse(readFileSync('package.json'));`.